src/tests/api-tests.ts

```diff
@@ -2135,13 +2135,16 @@ describe('api tests', () => {
       .build();
     await db.updateMicroblocks(microblock1);
 
+    // TODO: invalid test, the above function `db.updateMicroblocks` does not use the `microblock_canonical: false` property
```

Comment:
I'm not sure I understand. Should that have been provided at `updateMicroblocks`? IIRC I achieved this by setting `microblock_canonical: false` at the tx (line 2132).

Reply:
The `db.updateMicroblocks` function is provided data from the event observer (with a bit of massaging), and never encounters a `microblock_canonical == false` payload, so the re-org code never checks it. The test code above _does_ set `microblock_canonical`, but that field is not checked and is never false in practice.